src/main/java/com/example/moviemanager/service/MovieNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package com.example.moviemanager.service;
+
+public class MovieNotFoundException extends RuntimeException {
```

Comment:
Place it inside exception package. If there would be 10 service layer exceptions it would be hard to find service itself in service package.

Reply:
Moved into exception package.